dataeng/jobs/analytics/SnowflakeValidateStitch.groovy

```diff
@@ -0,0 +1,65 @@
+package analytics
+
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+
+
+class SnowflakeValidateStitch {
+    public static def job = { dslFactory, allVars ->
+        dslFactory.job('snowflake-validate-stitch') {
+
+            description(
+                'Validate a application database tables loaded by Stitch by comparing them against the same ' +
+                'tables loaded by Sqoop.  This compares only tables that exist in both sets, and only the last ' +
+                '10 days of changed rows.'
+            )
+
+            logRotator common_log_rotator(allVars)
+            parameters secure_scm_parameters(allVars)
+            parameters {
+                stringParam('ANALYTICS_TOOLS_URL', allVars.get('ANALYTICS_TOOLS_URL'), 'URL for the analytics tools repo.')
+                stringParam('ANALYTICS_TOOLS_BRANCH', allVars.get('ANALYTICS_TOOLS_BRANCH'), 'Branch of analytics tools repo to use.')
+                stringParam('APP_NAME', '', 'Application name of tables to validate.')
```

Comment:
Is the intent here to have a single Jenkins job that checks all four app DBs? Or four separate jobs? Where does `APP_NAME` come from here?

Reply:
Nm - just saw https://github.com/edx/jenkins-job-dsl/pull/1075

Reply:
yea, the intent is a single jenkins job to check all four app dbs.  That actually reminds me, I should update the build label to include the app name.

Reply:
If a single invocation of the `stitch_vs_sqoop_validation.py` is supposed to validate all four schemas, it'll need to be `--schema CREDENTIALS --schema LMS --schema ECOMMERCE --schema DISCOVERY`.

Reply:
That is, unless I change the script to work as `--schema CREDENTIALS LMS ECOMMERCE DISCOVERY`.